src/components/Query.js

```diff
@@ -0,0 +1,14 @@
+export default function Query({ inputValue, changeHandler, handleButtonClick }) {
+  return (
+    <>
+      <div>
+        <input
+          type="text"
+          value={inputValue}
+          onChange={(e) => changeHandler(e.target.value)}
```

Comment:
This confused me at first but im glad you understand your props passing and variable naming.